libkbfs/conflict_resolver.go

```diff
@@ -3043,6 +3066,12 @@ func (cr *ConflictResolver) completeResolution(ctx context.Context,
 		return err
 	}
 
+	defer func() {
```

Comment:
I think this is better put in syncBlocks right before the `doBlockPuts` call (per the comment on `doBlockPuts`)